jfuzzylite/src/main/java/com/fuzzylite/FuzzyLite.java

```diff
@@ -40,11 +42,14 @@
 
         @Override
         protected DecimalFormat initialValue() {
-            return new DecimalFormat("0.000");
+            NumberFormat df = NumberFormat.getNumberInstance(Locale.ROOT);
```

Comment:
I am not sure if we should use the root locale. I will look further into this, but I am more tempted to use the locale of the JVM, whatever that is. If we get the instance with the default locale, and the user changes the locale of the JVM, would the DecimalFormatter be updated automatically?